src/Core/Domain/Product/Exception/CannotUpdateProductException.php

```diff
@@ -50,4 +50,9 @@ class CannotUpdateProductException extends ProductException
      * When product tags update fails
      */
     const FAILED_UPDATE_TAGS = 40;
+
+    /**
+     * When failed to assign product categories
+     */
+    const FAILED_ASSIGN_TO_CATEGORIES = 50;
```

Comment:
```suggestion
    const FAILED_UPDATE_CATEGORIES = 50;
```
This should be renamed later for consistency, it can be done in another PR

Reply:
:+1: Ill better do it now :p